builtin/providers/postgresql/provider_test.go

```diff
@@ -0,0 +1,41 @@
+package postgresql
```

Comment:
Could you run `go fmt` on this file? Its formatting seems to be inconsistent with the other files and with Terraform's usual style.